codec/decoder/core/src/decode_slice.cpp

```diff
@@ -51,6 +51,13 @@
 #include "cpu_core.h"
 
 namespace WelsDec {
+//#define _MV_DUMP_ 1
+#ifdef _MV_DUMP_
+#if defined(_DEBUG)
+FILE* pFile = fopen ("Y:/p4/CodeProjects/Upwork/screencast/xiaotiansf/openh264-1/MV_CAVLC.txt", "w");
```

Comment:
I don't think this kind of leftover debug code should be included.

Reply:
I'll remove it as soon as I done.